saos-webapp/src/main/resources/message/common_pl.properties

```diff
@@ -133,6 +133,12 @@ judgmentSearch.results.paginationInfo = Pozycje od {0} do {1} spośród {2} wyni
 judgmentSearch.results.judgmentsNumber = {0,choice,0#0 orzeczeń|0<{0} orzeczenie|1<{0} orzeczenia|4<{0} orzeczeń} 
 judgmentSearch.results.link = Przejdź do strony szczegółów orzeczenia 
 
+judgmentSearch.hint.icon.alt = Ikona podpowiedzi - zaznaczenie spowoduje wyświetlenie tekstu podpowiedzi
+judgmentSearch.hint.reasons.title = Uzasadnienie podpowiedź
+judgmentSearch.hint.reasons.content = Typ uzasadnienie został stworzony, ponieważ dane z sądów powszechnych czasmi przychodza jako SENTENCE, a czasmi REASONS i nie da sie ch razem zmergować.
```

Comment:
Put 'orzeczenia.ms.gov.pl' to JudgmentSource.url.COMMON_COURT property and use it in jsp as argument to message 'judgmentSearch.hint.reasons.content'